docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.md

```diff
@@ -1,27 +1,60 @@
+{% if page.platform_type == "existing" %}
+## Прежде чем вы начнете
+
+Если вы развернули **новый** кластер и хотите поставить в него Deckhouse, то рекомендуем вернуться на предыдущий шаг, и использовать руководство по развертыванию кластера с помощью самого Deckhouse.  
+
+Данное руководство подразумевает установку Deckhouse в **минимальной** конфигурации, с включенным модулем [внутренней документации](../../documentation/v1/modules/810-deckhouse-web/) (для примера работы web-приложения), модулем [cert-manager](../../documentation/v1/modules/101-cert-manager/) и модулями, создающими некоторые CRD. 
+
+Использование минимальной конфигурации вместо конфигурации по умолчанию при установке в существующем кластере связано с тем, что в вашем кластере какой-то функционал может быть уже обеспечен другими приложениями. В таком случае, некоторые модули Deckhouse будет необходимо отключить. Например, если в кластере уже установлен CNI отличный от flannel, то в конфигурации Deckhouse необходимо отключить модуль [cni-flannel](../../documentation/v1/modules/035-cni-flannel/). Наверняка у вас уже есть Ingress-контроллер, и соответствующий модуль [ingress-nginx](../../documentation/v1/modules/402-ingress-nginx/) нужно также отключить. Некоторые установленные в кластере компоненты могут создавать ресурсы (например `CustomResourceDefinition`, `APIService`) которые создает Deckhouse, что может привести к конфликтам и т.п. 
```

Comment:
```suggestion
Использование минимальной конфигурации вместо конфигурации по умолчанию при установке в существующем кластере связано с тем, что в вашем кластере какой-то функционал может быть уже обеспечен другими приложениями. В таком случае, необходимо будет отключить некоторые модули Deckhouse. Например, если в кластере уже установлен CNI отличный от flannel, то в конфигурации Deckhouse необходимо отключить модуль [cni-flannel](../../documentation/v1/modules/035-cni-flannel/). Наверняка у вас уже есть Ingress-контроллер, и соответствующий модуль [ingress-nginx](../../documentation/v1/modules/402-ingress-nginx/) нужно также отключить. Некоторые установленные в кластере компоненты могут создавать ресурсы (например `CustomResourceDefinition`, `APIService`) которые создает Deckhouse, что может привести к конфликтам и т.п. 
```

Reply:
CNI flannel must be disabled, ingress can be enabled but there are some conditions when it can be harmful.